src/analysis/scoping_analysis.cpp

```diff
@@ -462,16 +490,27 @@ class NameCollectorVisitor : public ASTVisitor {
     bool visit_importfrom(AST_ImportFrom* node) override {
         for (int i = 0; i < node->names.size(); i++) {
             AST_alias* alias = node->names[i];
-            mangleNameInPlace(alias->asname, cur->private_name, scoping->getInternedStrings());
-            mangleNameInPlace(alias->name, cur->private_name, scoping->getInternedStrings());
-            if (alias->asname.str().size())
-                doWrite(alias->asname);
-            else
-                doWrite(alias->name);
+            if (alias->name.str() == std::string("*")) {
+                doImportStar(node);
```

Comment:
super random, but it looks like the name mangling should still happen on this path:
```python
class C(object):
    from __foo import *
```
throws `No module named _C__foo`